__tests__/updater-builder-integration.test.ts

```diff
@@ -91,5 +91,5 @@ integration('UpdaterBuilder', () => {
 
     await proxy.shutdown()
     await container.remove()
-  })
+  }, 30000)
```

Comment:
This was supposed to be a test, I forgot we have auto-merge on this repo... I reduced it in:
* https://github.com/github/dependabot-action/pull/1362